core/src/commonMain/kotlin/com/ustadmobile/core/io/ext/UmAppDatabaseContainerIoExt.kt

```diff
@@ -3,16 +3,22 @@ package com.ustadmobile.core.io.ext
 import com.ustadmobile.core.container.ContainerAddOptions
 import com.ustadmobile.core.db.UmAppDatabase
 import com.ustadmobile.door.DoorUri
+import org.kodein.di.DI
 
 expect suspend fun UmAppDatabase.addEntriesToContainerFromZip(containerUid: Long,
                                                               zipUri: DoorUri,
                                                               addOptions: ContainerAddOptions,
                                                               context: Any)
 
 expect suspend fun UmAppDatabase.addDirToContainer(containerUid: Long, dirUri: DoorUri,
-                                                   recursive: Boolean = true,
+                                                   recursive: Boolean = true, context: Any, di: DI,
                                                    addOptions: ContainerAddOptions)
 
 expect suspend fun UmAppDatabase.addFileToContainer(containerUid: Long, fileUri: DoorUri,
-                                                    pathInContainer: String, addOptions: ContainerAddOptions)
+                                                    pathInContainer: String, context: Any, di: DI, addOptions: ContainerAddOptions)
 
+
+expect suspend fun UmAppDatabase.addTorrentFileFromContainer(containerUid: Long,
```

Comment:
We're not really adding a torrent file here like the others, I think it would be better to call this writeContainerTorrentFile